lib/hubspot/company.rb

```diff
@@ -165,6 +165,18 @@ def add_contact!(company_vid, contact_vid)
                                      },
                                      body: nil)
       end
+
+      # Updates the properties of a company
+      # {http://developers.hubspot.com/docs/methods/companies/update_company}
+      # @param vid [Integer] hubspot company vid
+      # @param params [Hash] hash of properties to update
+      # @return [Hubspot::Company] Company record
+      def update!(vid, params)
```

Comment:
This is a bit trivial but what do you think of calling this method `update` without the `!`? I tend to avoid the use of `!` unless there is a safe version of the function. For example, `stringify_keys!` has the alternative `stringify_keys`.

I find that avoiding defining bang (!) methods leads to more descriptive function names. Since `!` is a rather debated topic, I find it's use doesn't hold much value, except in the event where `!` means there's an alternative option. 

Better Naming example: Sending an invite email to a user. Instead of writing `invite!` to warn the reader something will happen, `send_invite` is a clearer message that informs the reader that an invite will be sent.

Reply:
Other classes in the gem seem to use the `!` on persistence style method calls. I was assuming this was following the ActiveRecord convention of bang versions of persistence methods raising exceptions whereas the non-bang version returns true/false.